distribution/docs/src/main/jdocs/content/_security-framework/security-core-api-contents.adoc

```diff
@@ -0,0 +1,18 @@
+:title: Security Core API
+:type: subSecurityFramework
+:status: published
+:parent: Security Core
+:order: 00
+:summary: Security Core API.
+
+The Security Core API contains all of the ${branding} Security Framework APIs that are used to perform security operations within ${branding}.
+
+===== Installing the Security Core API
+
+The Security Services App installs this bundle by default.
```

Comment:
Replace the pronoun "this" with "the Security Core API"

Reply:
done.